### PR TITLE
upgrade to latest esdoc

### PR DESCRIPTION
**Pull request by icambron**
There have been a ton of changes to ESDoc, so I thought I'd update this tool.

### CODE REVIEW

**Comment by fkm** [comment]
I guess 'tis the season ;-)

Let's hope this will be merged after the holidays.

**Comment by jwgwalton** [comment]
Is anyone available to review this? Cheers

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "should": "^11.1.2"
   },
   "dependencies": {
-    "esdoc": "^0.5.2",
```
Comment (fkm) [inline package.json:35]: Wouldn't it be better to use `^1.0.1`?
It's the earliest version that should work with the Gulp plugin.

https://github.com/esdoc/esdoc/blob/master/CHANGELOG.md#101-2017-07-30

*But, I honestly only tested with 1.0.4 myself!*

**Comment by fkm** [comment]
Not sure if a review is necessary. But, I added one :-)

**Review by fkm — COMMENTED** [review]
Not sure if a review is what was missing.
But, here you go :-)

File: package.json (hunk 1 above)
Comment (icambron) [reply package.json:35]: Same reason here. It's just what I tested so it's what I put in.

**Comment by fkm** [comment]
@nanopx Could you please merge the request or make the necessary changes on your own?

**Pull request closed without merging**